### ISSUE TITLE
Module not found

### ISSUE THREAD

**Issue by mikestaub** [jspm/generator#83]
How do can I resolve these errors?

```
JspmError: Module not found: https://ga.jspm.io/npm:yaml@1.10.2/browser/index.js imported from https://ga.jspm.io/npm:cosmiconfig@6.0.0/dist/index.js
```

**Comment by guybedford** [comment]
Thanks @mikestaub for posting the issue here. There was a build error on yaml which I've resolved, it should be working now. Let me know if you're still having issues.

**Comment by mikestaub** [comment]
Thanks for the quick fix! I don't see a new npm version published ( 1.0.0-beta.18 ) should I just try pulling master down?

**Comment by guybedford** [comment]
It's the CDN that's been fixed. So it should just work now. If not let me know what error you are getting.
